backend/core/settings/prod.py

```
@@ -0,0 +1,64 @@
+"""Production ready settings for the Apatie backend project."""
+
+from __future__ import annotations
+
+import secrets
+
+from django.core.exceptions import ImproperlyConfigured
+
+from .base import *  # noqa: F401,F403
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)